mlir/include/mlir-c/Dialect/SparseTensor.h

```diff
@@ -25,6 +25,8 @@ MLIR_DECLARE_CAPI_DIALECT_REGISTRATION(SparseTensor, sparse_tensor);
 /// These correspond to SparseTensorEncodingAttr::LevelType in the C++ API.
 /// If updating, keep them in sync and update the static_assert in the impl
 /// file.
+typedef uint64_t MlirBaseLevelType;
```

Comment:
I think this should be MlirSparseTensorLevelType
(i.e. an entity that really represents a level type, properties and format)

Reply:
you mean the typedef after I switch the name right?